randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -482,7 +492,8 @@ def create_data(self) -> dict:
                 for pickup_item in pickup_list
                 if (data := self._pickup_detail_for_target(pickup_item)) is not None
             ],
-            "elevators": [],
+            # FIXME: keep elevators empty in vanilla mode in case something is broken with patching elevators
```

Comment:
this probably doesn't need a FIXME. this is a pretty typical pattern in patch data factories